src/app/weak-supervision/components/weak-source-details/weak-source-details.component.ts

```diff
@@ -424,7 +424,7 @@ export class WeakSourceDetailsComponent
           this.description,
           functionName
         ).pipe(first())
-        .subscribe();
+        .subscribe(() => this.informationSourceQuery$.refetch());
```

Comment:
shouldn't be necessary because of the websockets (or at least that's the theory, ill double check)

Reply:
it was only updated for different user ids (don't know why though).

Also refetching by hand on mutations isn't a common approach, usually, we would use refetch queries in apollo (however it's already there so no clue why it didn't work).

Can be resolved.